src/platforms/javascript/guides/nextjs/manual-setup/index.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Manual Setup
+sidebar_order: 2
+description: "Review the options to manage."
```

Comment:
I see this is the same as `src/platforms/react-native/manual-setup/index.mdx`.

@PeloWriter would you have some suggestion of a better description structure to improve SEO?

Reply:
The description content for this page doesn't display, as the index page isn't pulled into a PageGrid. Nonetheless, I agree, that isn't terribly useful. 
```suggestion
description: "Learn more about manually configuration, including webpack usage and managing your app's property file."
```